### PR TITLE
Buttons added

### PR DESCRIPTION
**Pull request by NerdyGent**
Page Details should be finished!

### CODE REVIEW

File: src/app/pages/place-details/place-details.page.html [hunk 1]
```diff
@@ -8,243 +8,128 @@
...
+                      color="light" name="heart-outline"></ion-icon><span class="btnText">Save</span></ion-button>
+
+                  <ion-button *ngIf="this.userSavedPlace" (click)="removePlacefromMyPlaces()"
+                    class="changeBtn MobileButtons ion-text-uppercase ion-margin-end" color="secondary"><ion-icon
+                      color="light" name="close-circle-outline"></ion-icon><span
+                      class="btnText">Saved</span></ion-button>
+
+                  <ion-button *ngIf="!this.userSavedPlace" (click)="toggleVisited()"
+                    class="changeBtn MobileButtons ion-text-uppercase " color="primary"><ion-icon color="light"
+                      name="add-circle"></ion-icon><span class="btnText MobileButtons">Visit</span></ion-button>
```
Comment (kendall-mchenry) [inline src/app/pages/place-details/place-details.page.html]: I was thinking we could update the visited icons to be more clear. 
Unvisited = pin outline
Visited = checkmark

So here, the icon would be the pin-outline

File: src/app/pages/place-details/place-details.page.html [hunk 2]
```diff
@@ -8,243 +8,128 @@
...
+
+                  <ion-button *ngIf="this.userSavedPlace && !this.currentMyPlace.visited" (click)="toggleVisited()"
+                    class="changeBtn ion-text-uppercase MobileButtons" color="primary"><ion-icon color="light"
+                      name="add-circle"></ion-icon><span class="btnTex MobileButtons">Visit</span></ion-button>
```
Comment (kendall-mchenry) [inline src/app/pages/place-details/place-details.page.html]: Here, icon would be pin-outline

File: src/app/pages/place-details/place-details.page.html [hunk 3]
```diff
@@ -8,243 +8,128 @@
...
+
+                  <ion-button *ngIf="this.userSavedPlace && this.currentMyPlace.visited" (click)="toggleVisited()"
+                    class="changeBtn ion-text-uppercase MobileButtons" color="secondary"><ion-icon color="light"
+                      name="pin-outline"></ion-icon><span class="btnText MobileButtons">Visited</span>
```
Comment (kendall-mchenry) [inline src/app/pages/place-details/place-details.page.html]: Here, icon would be checkmark

**Review by kendall-mchenry — COMMENTED** [review]
Just a note about the visited icons. I was thinking that would help with my peace about using the icons since they're a little more clear.
Unvisited = pin-outline
Visited = checkmark

File: src/app/pages/place-details/place-details.page.html (hunk 1 above)
Comment (NerdyGent) [reply src/app/pages/place-details/place-details.page.html]: I'll make the changes in a new PR. Hold on.

**Pull request merged**